docs/index.rst

```diff
@@ -186,11 +186,28 @@ For ``POST`` and ``PATCH`` requests, if a client provides read-only fields, the
 Concepts
 ~~~~~~~~
 
+.. _concepts-users:
+
 Users
 *****
-Users have access to a set of :ref:`projects <concepts-projects>`. Depending on their :ref:`permissions <permissions>`, users can view or modify a project and its :ref:`dialogues <concepts-dialogues>`.
+Users have access to a set of :ref:`projects <concepts-projects>` based on the permisions of the :ref:`teams <concepts-teams>` they are members of. The projects that users have access to may span multiple :ref:`organizations <concepts-orgs>`.
+
+.. _concepts-teams:
+
+Teams
+*****
+A team comprises a set of :ref:`users <concepts-users>`. Each team has a set of :ref:`permissions <permissions>` defining the actions its members are permitted to carry out.
 
-.. TODO link to auth once this is documented.
+.. _concepts-orgs:
+
+Organizations
+*************
+Organisations comprise a set of :ref:`projects <concepts-projects>`, :ref:`teams <concepts-teams>` and :ref:`user <concepts-users>`\s.
```

Comment:
Sphinx gets confused if the character following a closing backtick isn't escaped. But reading over this line again, it looks like the `:ref:` should be `users` anyways, so it isn't needed, will fix.